docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -11,20 +11,27 @@ process, definitely before acceptance testing starts, but also before
 development has been completed.
 
 
-## 2. [Create test distributions]
+## 2. [Test planning]
+
+This phase should be performed after the end of development but before testing
+process, definitely before acceptance testing starts, but also before
+development has been completed.
```

Comment:
"after the end of development" and "before development has been completed". Should it be done twice? If so, state it in the right order, and in two different bullets.

Reply:
Something is very wrong here. I'll make an update.

Reply:
I fixed it.